packages/contracts-bedrock/deploy-config/devnetL1-template.json

```diff
@@ -49,6 +49,7 @@
   "l2GenesisEcotoneTimeOffset": "0x0",
   "l2GenesisFjordTimeOffset": "0x0",
   "l2GenesisGraniteTimeOffset": "0x0",
+  "l2GenesisBlobTimeOffset": "0xffffffff",
```

Comment:
Why is it set to `0xffffffff`?

Reply:
It means it is disabled by default.  The reason is that we cannot specify a da server in devnet (see below) so the feature is disabled by default.  Otherwise, `op-node` will fail to start if l2blob is on, while no da server is specified.